lib/ui/discover/discover_child.dart

```diff
@@ -1,14 +1,47 @@
+import 'package:farmsmart_flutter/model/loading_status.dart';
+import 'package:farmsmart_flutter/redux/app/app_state.dart';
 import 'package:farmsmart_flutter/utils/assets.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:farmsmart_flutter/data/model/article_entity.dart';
+import 'package:farmsmart_flutter/ui/discover/discover_child_item.dart';
+import 'package:farmsmart_flutter/ui/discover/discover_viewmodel.dart';
+import 'package:farmsmart_flutter/redux/home/discover/discover_actions.dart';
 
-class HomeDiscoverChild extends StatelessWidget {
-  HomeDiscoverChild();
 
+class HomeDiscoverChild extends StatefulWidget {
```

Comment:
I don't quite get the name HomeDiscoverChild.  It looks to be the view articles page but I got confused on the use of Home and Child.

Reply:
Seems to be the same approach as myplot.

From what I understand: 
child -> same concept as page
detail screen -> the actual detail screen of the child

In my understanding child and detail can be the same thing but it's different here. The child is actually the parent screen.
Is my understanding correct?

Reply:
Child is referred to the bottomBar, so myPlot, P/L, Discover and Community are every child of the bottom bar.
Item is every element listed.
Detail is the screen inside every item in the list

But you are right with the naming,i think is a little bit confusing so we can rename all the children of bottom bar to Page, like:

 HomeDiscoverChild => HomeDiscoverPage,
 and HomeMyPlotChild => HomeMyPlotPage

Thnks!

Reply:
Thanks guys!